### PR TITLE
EDSC-1419 Bring back 2d coordinate search

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

**Comment by mreese84** [comment]
@mightynimble, upon initial view of the filter stack work... I'd add a little padding between the end of the value input field and the "x" icon. I'd also make the color of the help text grey so it blends in a little. Seems too harsh/in-your-face as it is. Maybe even bump the font size down a size or two.

**Comment by mreese84** [comment]
Also, looks like there's a merge conflict that needs to get knocked out.

**Comment by mreese84** [comment]
@mightynimble, also noticed that the value input field is so long that it is stretching out the "clear filters" icon box. Notice in the screenshot that the clear filters button is longer than the other 2 icons. We should shorten the value input field so that it doesn't stretch out the icon.

screen shot 2017-05-01 at 9 32 11 am

File: app/assets/stylesheets/modules/map/map.css.scss [hunk 1]
```diff
@@ -306,6 +306,32 @@ p.map-popup-info {
...
+    height: 22px;
+  }
+
+  label {
+    min-width: 0;
+  }
+}
+
+.landing-toolbar-content .error {
+  max-width: 590px;
```
Comment (mightynimble) [inline app/assets/stylesheets/modules/map/map.css.scss]: This is more like a hack to me - is there a place that defines the max width of the filter stack? @mreese84

**Comment by mightynimble** [comment]
Pushed css changes that you mentioned, @mreese84 . Also see my comment above.

**Comment by mightynimble** [comment]
Right now I am trying to find a way to style the 2d coord input to fill the remaining space of the container. If you know something off the top of your head, I am very appreciated to learn :)

screen shot 2017-05-01 at 10 42 22 am

**Pull request merged**